src/index.html

```diff
@@ -4,9 +4,342 @@
+      </nav>
+    </header>
+
+    <main class="main">
+
+      <div class="card" data-qa="card">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="Product photo"
+        >
+
+        <h2 class="card__descr">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-descr">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-descr">$2,199</p>
+        </div>
+
+        <a
+          class="card__button"
+          href="#"
+          data-qa="card-hover"
+        >
+        Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="Product photo"
+        >
+
+        <h2 class="card__descr">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-descr">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-descr">$2,199</p>
+        </div>
+
+        <a
+          class="card__button"
+          href="#"
+        >
+        Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="Product photo"
+        >
+
+        <h2 class="card__descr">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-descr">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-descr">$2,199</p>
+        </div>
+
+        <a
+          class="card__button"
+          href="#"
+        >
+        Buy
+        </a>
```

Comment:
Fix it everywhere if you have only two attributes
```suggestion
        <a class="card__button" href="#">
          Buy
        </a>
```